Configs/.config/hyprdots/scripts/quickapps.sh

```diff
@@ -60,9 +60,15 @@ r_override="window{height:${dockHeight};width:${dockWidth};border-radius:${wind_
 
 # launch rofi menu
 
+if [ -d /run/current-system/sw/share/applications ]; then
+    appDir=/run/current-system/sw/share/applications
+else
+    appDir=/usr/share/applications
+fi
+
 RofiSel=$( for qapp in "$@"
 do
-    Lkp=`grep "$qapp" /usr/share/applications/* | grep 'Exec=' | awk -F ':' '{print $1}' | head -1`
+    Lkp=`grep "$qapp" $appDir/applications/* | grep 'Exec=' | awk -F ':' '{print $1}' | head -1`
```

Comment:
here the `$appDir` already points to `/usr/share/applications`, so please remove `applications` from this line

```
Lkp=`grep "$qapp" $appDir/* | grep 'Exec=' | awk -F ':' '{print $1}' | head -1`
```